recipes/zeus_expected/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.cmake import cmake_layout
+import os
+
+required_conan_version = ">=2.0"
```

Comment:
```suggestion
required_conan_version = ">=1.54.0"
```

Conan 1.x is still mandatory.

Reply:
Nice catch @uilianries I totally glossed over it :)

Reply:
> Do you mean if the CI run both? 

No, I mean personal development env. I managed to do it by create a new user for conan 1.x testing only. Just want to know if there are any eaiser way.

Reply:
Ah, now I see. Well, you have some options, depending what you prefer:

- You could use Docker images to validate linux builds:

```
cd conan-center-index/recipes/
docker pull conanio/gcc11-ubuntu16.04:1.64.0
docker run -ti -v $PWD:/home/conan/project conanio/gcc11-ubuntu16.04:1.64.0
# In the container
cd project/zeus_expected/
conan create all 1.0.2@
```

Note that I passed the docker tag as the Conan version. Avoid the tag `latest` always.
All images are hosted in hub.docker.com/r/conanio, but only of few are maintained/used nowadays: https://github.com/conan-io/conan-docker-tools?tab=readme-ov-file#storage-and-availability

- Use [pyenv](https://github.com/pyenv/pyenv) to create separated environments (I use this one in Mac)

```
pyenv virtualenv conan1
pyenv activate conan1
pip install conan==1.64.0
cd conan-center-index/recipes/zeus_expected/
conan create all 1.0.2@
...
pyenv deactivate conan1
```

The environment `conan1` will be always available, so you can activate again. You can do the same for Conan 2.x in case wanting to have more versions installed.